Automattic/jetpack
# docker: Add node and pnpm to the dev environment

#### Changes proposed in this Pull Request:

This may make it easier to develop inside the environment, at the cost
of an extra 107MB in image size (95 for Node and 12 for pnpm).

I don't know whether we actually want it at that cost, so putting this up for review separately.

#### Jetpack product discussion


None

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Build the image with `jetpack docker build-image`.
* Inspect the container, e.g. by doing `docker run --rm -it --entrypoint=bash automattic/jetpack-wordpress-dev:latest`. Check that node, pnpm, and pnpx are installed at correct versions.
